package.json

```diff
@@ -67,11 +71,22 @@
         "parserOptions": {
             "parser": "babel-eslint"
         },
-        "rules": {}
+        "rules": {},
+        "overrides": [
+            {
+                "files": [
+                "**/__tests__/*.{j,t}s?(x)",
+                "**/tests/unit/**/*.spec.{j,t}s?(x)"
+                ],
+                "env": {
+                "jest": true
+                }
```

Comment:
can you explain this a bit?

Reply:
These are the default configs that comes in with jest. This came in after I ran this command:
```
vue add unit-jest
```